### DISCUSSION TITLE
cat with prey (mouse and birds in mouth)

### DISCUSSION THREAD

**Discussion by kjetilhelge** [blakeblackshear/frigate#14335] [General]
I see that both cat and mouse ar detectable objects, but can i use frigate to detect if my cat wants to bring me gifts as in dead/alive mice and birds? If not, I would love if that was an upcoming feature. Planning to lock the car dorr when prey is detected.

**Comment by NickM-27** [comment]
`mouse` refers to computer mouse not the animal. And no, there's no easy way to specifically detect if an object is carrying another object

**Reply by kjetilhelge** [reply]
Ah, I just assumed it was the animal. I've seen some models on the roboflow site that contains hundreds to thousands of images with cats carrying a mouse, but maybe those models are in a different format than what friagate use. I just started looking for a solution to my cat/mouse problem, so I have to keep searching and learning.

**Reply by htilly** [reply]
@kjetilhelge 
Did you ever get this to work?!
Im trying to solve the same issue. Frigate+ might be an option? 

I have a Coral Edge TPU attached today, but more than willing to alter the setup to get this working =)

**Reply by kjetilhelge** [reply]
Unfortunatly not I'm afraid. I didn't her frigate to work properly with just detecting cats, so I pur the project om ice... Maybe I pick it up again after summer. I have also looked into other devices like Luckfox pro Max with camera, and might give that a try.

**Reply by ehn** [reply]
Frigate+ now has `rodent` as a candidate label.
